### ISSUE TITLE
Add idle_timeout and running_timeout to google_workstations_workstation_config resource

### ISSUE THREAD

**Issue by daanheikens** [hashicorp/terraform-provider-google#14179]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment. If the issue is assigned to the "modular-magician" user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If the issue is assigned to a user, that user is claiming responsibility for the issue. If the issue is assigned to "hashibot", a community member has claimed the issue already.

### Description

According to the Google Cloud Workstation API reference, there are two fields in the WorkstationConfig object to control the lifetime of a Cloud Workstation. Idle_timeout and running_timeout. idle_timeout defines when the workstation will be shut down when it's idle, therefore only charging when the workstation is actually in use. running_timeout defines how long to wait before automatically stopping a workstation after it started.

Currently, these two fields are missing from the related Terraform resource.

Documentation: https://cloud.google.com/workstations/docs/reference/rpc/google.cloud.workstations.v1beta#google.cloud.workstations.v1beta.WorkstationConfig

### New or Affected Resource(s)

* google_workstations_workstation_config

### Potential Terraform Configuration

```tf
resource "google_workstations_workstation_config" "default" {
  provider = google-beta
  workstation_config_id = "workstation-config"
  workstation_cluster_id = google_workstations_workstation_cluster.default.workstation_cluster_id
  location = "us-central1"
  idle_timeout = 1200 // 20 minutes in seconds
  running_timeout = 43200 // 12 hours in seconds

  host {
    gce_instance {
      machine_type                = "e2-standard-4"
      boot_disk_size_gb           = 35
      disable_public_ip_addresses = true
    }
  }
}
```

**Comment by bschaatsbergen** [comment]
Thanks for taking the time to raise this issue @daanheikens 🙌🏼 , I'll be picking this up as soon as possible.

**Comment by mcinquin** [comment]
Hello,

Do you know when this setting will be released?

Regards.

**Comment by davidalger** [comment]
Looks like this was released in v4.68.0 last week: https://github.com/hashicorp/terraform-provider-google-beta/releases/tag/v4.68.0 🎉

**Comment by bschaatsbergen** [comment]
Amazing! cc @daanheikens
